### ISSUE TITLE
Import feature? 

### ISSUE THREAD

**Issue by erikjanwestendorp** [dawoe/umbraco-redirectsviewer#25]
First of all, what a great package!

Are there plans to expand the package with an import option, for example csv? Many of our customers deliver the redirects that way. Would be great if there was a possibility to import a lot of redirects at once.

**Comment by dawoe** [comment]
Hi @erikjanwestendorp 

This was never the intention of this package. The first version I built was because a client wanted to see the redirects for a node without searching on the dashboard. And quickly they requested add and delete functionality.

I also don't think the built in umbraco redirect table is suited for importing redirects. Personally I would go with redirects at IIS level or use another redirect tool.

If you want a import for the umbraco table you can look at the code in this [PR ](https://github.com/dawoe/umbraco-redirectsviewer/pull/11) that I did not merge. It can be a starting point.

Dave

**Comment by erikjanwestendorp** [comment]
@dawoe I think you are right that importing redirects is not the best idea and it is better to do this at the IIS level. Is it an idea to display the IIS redirects on a node?

Thanks for the reference to PR 😄

**Comment by dawoe** [comment]
Need to think about how to do that.  Perhaps provide a extension point so you could do it your self.  Like an event where you can you can add your own urls to the table. Or implement a interface.

The reason is that IIS can have many redirects that match every url... eg http to https, non www to www.... etc. So that can get messy.

Dave

**Comment by erikjanwestendorp** [comment]
@dawoe Yes an interface (or event) would be great so that everyone using your package can decide what to show in the table. I think it should be a read only record where deletions are not supported.
